lib/pure/os.nim

```diff
@@ -1458,16 +1466,13 @@ proc normalizePath*(path: var string) {.rtl, extern: "nos$1", tags: [].} =
 
 proc normalizePathAux(path: var string) = normalizePath(path)
 
-proc normalizedPath*(path: string): string {.rtl, extern: "nos$1", tags: [].} =
-  ## Returns a normalized path for the current OS.
-  ##
-  ## See also:
-  ## * `absolutePath proc <#absolutePath,string>`_
-  ## * `normalizePath proc <#normalizePath,string>`_ for the in-place version
-  runnableExamples:
-    when defined(posix):
-      assert normalizedPath("a///b//..//c///d") == "a/c/d"
-  result = pathnorm.normalizePath(path)
+proc normalizedPath*(path: string, exe = false): string {.rtl, extern: "nos$1", tags: [].} =
+  ## In-place version of `normalizePath proc <#normalizePath,string>`_
```

Comment:
You're calling this the "outplace" version above, which is it? According to the prior description this is wrong. Just restore the old one.

Reply:
was a typo, fixed inplace => outplace